src/Invariants/zero_forcing/minimum_zero_forcing_set.jl

```diff
@@ -3,21 +3,114 @@
 
 Return a minimum zero forcing set of `g`.
 
+"""
+# previous implementation: what should we do with this?
+
+# function compute(
+#     ::Type{MinimumZeroForcingSet},
+#     g::SimpleGraph{T}
+# ) where T <: Integer
+
+#     # Get the number of vertices in `g`.
+#     n = Graphs.nv(g)
+
+#     for size in 1:n
+#         for subset in Combinatorics.combinations(1:n, size)
+#             blue = Set(subset)
+#             apply!(ZeroForcingRule, blue, g; max_iter=n)
+#             length(blue) == n && return MinimumZeroForcingSet(subset)
+#         end
+#     end
+#     error("Could not find a minimum zero forcing set for $g")
+# end
+
+"""
+    compute(::Type{MinimumZeroForcingSet}, g::SimpleGraph)
+
+Return a minimum zero forcing set of undirected graph `g` using Integer Programming
+formulation from Brimkov et al. (2019), published in EJOR:
+https://www.sciencedirect.com/science/article/pii/S0377221718308063#sec0007
 """
 function compute(
     ::Type{MinimumZeroForcingSet},
     g::SimpleGraph{T}
 ) where T <: Integer
 
-    # Get the number of vertices in `g`.
+    # convert to directed graph, replacing each edge with two directed edges
+    directed_g = Graphs.DiGraph(g)
+
+    return _compute(MinimumZeroForcingSet, directed_g)
+end
+
+# helper function for computing minimum zero forcing set of undirected graph...
+# if it is already directed, can we just use this function? (and rename + document it)
```

Comment:
Yeah, I think so

Reply:
I'll address this comment and the previous one in my next commit. I wanted to push the bug fixes first before changing anything else.